mix.exs

```diff
@@ -86,14 +70,12 @@ defmodule Chat.MixProject do
   # See the documentation for `Mix` for more info on aliases.
   defp aliases do
     [
-      "assets.deploy": ["tailwind default --minify", "esbuild default --minify", "phx.digest"],
-      c: ["coveralls.html"],
```

Comment:
I think we can keep this command

Reply:
`mix c`?  I agree. I created the project from scratch using Phoenix 1.7 generator, so custom commands were defo overridden. 👍

Reply:
Restored _manually_ on the branch. 

```sh
n@mbp16 phoenix-chat-example % mix c
............
Finished in 0.5 seconds (0.5s async, 0.04s sync)
12 tests, 0 failures

Randomized with seed 414798
----------------
COV    FILE                                        LINES RELEVANT   MISSED
100.0% lib/chat.ex                                     9        0        0
100.0% lib/chat/message.ex                            26        4        0
100.0% lib/chat/repo.ex                                5        0        0
100.0% lib/chat_web/channels/room_channel.ex          51       11        0
100.0% lib/chat_web/components/layouts.ex              5        0        0
100.0% lib/chat_web/controllers/auth_controller       14        2        0
100.0% lib/chat_web/controllers/error_html.ex         19        1        0
100.0% lib/chat_web/controllers/error_json.ex         15        1        0
100.0% lib/chat_web/controllers/page_controller        9        1        0
100.0% lib/chat_web/controllers/page_html.ex           9        1        0
100.0% lib/chat_web/endpoint.ex                       49        0        0
100.0% lib/chat_web/router.ex                         32        5        0
[TOTAL] 100.0%
----------------
Generating report...
```